Polygon_mesh_processing/test/Polygon_mesh_processing/test_pmp_clip.cpp

```diff
@@ -462,15 +465,80 @@ void test()
   }
 }
 
+template <class Mesh>
+void test_split_plane()
+{
+  // test with a clipper mesh
+  Mesh tm1;
+
+  std::ifstream input("data-coref/elephant.off");
+  input >> tm1;
+  input.close();
+
+  PMP::split(tm1,K::Plane_3(0,0,1,0));
+  std::vector<Mesh> meshes;
+  PMP::split_connected_components(tm1,
+                                  meshes,
+                                  params::all_default());
+  //CGAL_assertion(meshes.size() == 3);
```

Comment:
@maxGimeno Should this be uncommented?

Reply:
I'll look again to the tests and see about that